src/app/containers/ChartbeatAnalytics/index.jsx

```diff
@@ -41,6 +41,7 @@ const ChartbeatAnalytics = ({ data }) => {
     service,
     sendCanonicalChartbeatBeacon,
     isCanonicalAndEnabled,
+    chartbeatConfig,
```

Comment:
I've seen this added, then removed in a few PRs, could maybe bundle this into the PR for https://github.com/bbc/simorgh/issues/4811 , that way we will test chartbeat after this change is made? We would not expect to be testing chartbeat for this PR.

Reply:
The reason I actually added this was because eslint complains even though I didn't touch that code.

```
React Hook useEffect has a missing dependency: 'chartbeatConfig'. Either include it or remove the dependency array  react-hooks/exhaustive-deps
```

Reply:
Does that prevent you committing due to errors from the pre-commit hook?

Reply:
No it doesn't. It's just a warning. Updated.

Reply:
thanks @samora I'm gonna make a note on #4811 to include this change